cmd/utils/flags_legacy.go

```diff
@@ -38,8 +38,6 @@ var DeprecatedFlags = []cli.Flag{
 	LegacyLightServFlag,
 	LegacyLightPeersFlag,
 	LegacyMinerThreadsFlag,
-	LegacyMinerGasTargetFlag,
-	LegacyMinerGasPriceFlag,
```

Comment:
If we're keeping the gas price flag, it seems like we may want to keep the legacy version.

Reply:
moved `--miner.gasprice` to deprecated and removed the already deprecated one